apps/aecore/src/aec_peers.erl

```diff
@@ -685,6 +683,15 @@ parse_uri(Uri) ->
             Error
     end.
 
+uri_safe_parse(Uri) when is_binary(Uri) -> %% Workaround for https://github.com/erlang/otp/pull/1739
+    uri_safe_parse(unicode:characters_to_list(Uri, utf8));
+uri_safe_parse(Uri) ->
+    F = aeu_uri:scheme_validation_fun(valid_uri_schemes()),
+    http_uri:parse(Uri, [{scheme_validation_fun, F}]).
+
+valid_uri_schemes() ->
```

Comment:
would it make sense to also add HTTP capitalized?

Reply:
I pushed a commit meant to address this, CI is running.